docs/sources/source/nlp/dialogue_state_tracking.rst

```diff
@@ -264,3 +263,124 @@ References
     :style: plain
     :labelprefix: NLP-DST
     :keyprefix: nlp-dst-
+
+
+
+SGD Tutorial
+============
+
+
+Introduction
+------------
+
+One of the main building blocks of a task-oriented dialogue system that can book a table in a restaurant or order food for you is a Dialogue State Tracker (DST).
+DST should not only understand what the user just said but also remember what was said before.
+DST carries the information about what intent the user has in the conversation, for example, find a restaurant or book a plane ticket,
+and what slots along with the corresponding values were mentioned in the dialogue.
+
+
+
+This tutorial is based on the code from `examples/nlp/dialogue_state_tracking/dialogue_state_tracking_sgd.py  <https://github.com/NVIDIA/NeMo/blob/master/examples/nlp/dialogue_state_tracking/dialogue_state_tracking_sgd.py>`_.
+
+
+The Schema-Guided Dialogue Dataset
+----------------------------------
+
+In this tutorial, we are using the Schema-Guided Dialogue (SGD) dataset :cite:`nlp-sgd-rastogi2019towards` that contains over 16k multi-domain goal-oriented conversations across 16 domains.
+The data represents conversations between a user and a virtual assistant, and it can be used for various dialogue management tasks:
+intent prediction, slot filling, dialogue state tracking, policy imitation learning, language generation. 
+
+One part of the dialogues in the dataset spans across only a single domain, use ``--task_name dstc8_single_domain`` to use such dialogues. Another part focuses only on dialogues that span across multiple domains during a single conversation, 
```

Comment:
Ok, I do not get it... How can the user indicate a particular domain? Should the user put specific domain name (restaurant) instead of `dstc8_single_domain` ?

Reply:
there is no option to select a particular domain, just examples that have only a single domain in the dialogue

Reply:
I guess we would need that feature to dissect more our models and accuracy/their results.

Anyway, not important now...